internal/controller/numaflowcontrollerrollout_controller_test.go

```diff
@@ -65,6 +68,23 @@ var _ = Describe("NumaflowControllerRollout Controller", func() {
 			By("Cleanup the specific resource instance NumaflowControllerRollout")
 			Expect(k8sClient.Delete(ctx, resource)).To(Succeed())
 		})
+
+		It("Should throw a CR validation error", func() {
```

Comment:
its trivial, but do you mind adding another test to ensure the creation of second rollout object with numaflow-controller won't succeed. Thanks!